bec_widgets/widgets/plots/waveform/waveform.py

```diff
@@ -1529,13 +1529,18 @@ def _check_async_signal_found(self, name: str, signal: str) -> bool:
             signal(str): The entry of the async signal.
 
         Returns:
-            bool: True if the async signal is found, False otherwise.
+            tuple[bool, str]: A tuple where the first element is True if the async signal is found (False otherwise),
+                and the second element is the signal name (either the original signal or the storage_name for AsyncMultiSignal).
         """
-        bec_async_signals = self.client.device_manager.get_bec_signals("AsyncSignal")
+        bec_async_signals = self.client.device_manager.get_bec_signals(
+            ["AsyncSignal", "AsyncMultiSignal"]
+        )
         for entry_name, _, entry_data in bec_async_signals:
             if entry_name == name and entry_data.get("obj_name") == signal:
-                return True
-        return False
+                if entry_data.get("signal_class") == "AsyncMultiSignal":
+                    return True, entry_data.get("storage_name")
```

Comment:
you can always take the storage name, not only for async multi signals. Any BEC Signal reports it

Reply:
They describe different things. If you want to retrieve the suffix for the data storage in redis, use the storage name. The obj_name describes the data entry within a read